docs/api/v3.rst

```diff
@@ -0,0 +1,558 @@
+            "default_version": "stable",
+            "default_branch": "master",
+            "documentation_type": "sphinx_htmldir",
+            "canonical_url": "http://pip.pypa.io/en/stable/",
+            "links": [
+                {
+                    "href": "/api/v3/project/pip/users/",
+                    "rel": "users",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/versions/",
+                    "rel": "versions",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/builds/",
+                    "rel": "builds",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/domains/",
+                    "rel": "domains",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/redirects/",
+                    "rel": "redirects",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/notifications/",
+                    "rel": "notifications",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/features/",
+                    "rel": "features",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/subprojects/",
+                    "rel": "subprojects",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/translations/",
+                    "rel": "translations",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/integrations/",
+                    "rel": "translations",
+                    "type": "GET"
+                }
+            ]
+        }
+
+
+    :>json string name: The name of the project.
+    :>json string slug: The project slug (used in the URL).
+    :>json string description: An RST description of the project
+    :>json string language: The language code of this project
+    :>json string programming_language: The programming language of the project (eg. "py", "js")
+    :>json string repo: The repository URL for the project
+    :>json string repo_type: Version control repository of the project
+    :>json string default_version: The default version of the project (eg. "latest", "stable", "v3")
+    :>json string default_branch: The default version control branch
+    :>json string documentation_type: An RST description of the project
+    :>json string canonical_url: The canonical URL of the default docs
+    :>json array links: Array with HEATEOAS_ links to retrieve related information
+
+    :statuscode 200: Success
+    :statuscode 404: There is no ``Project`` with this slug
```

Comment:
We could add a top level section with all the status code and what they return (at the beginning it says we always return a json)